packages/app/src/app/pages/Dashboard/Components/Sandbox/NewSandbox.tsx

```diff
@@ -21,18 +21,19 @@ export const NewSandbox: React.FC<NewSandboxProps> = props => {
         fontSize: 3,
         backgroundColor: 'grays.700',
         border: '1px solid',
-        borderColor: 'grays.600',
+        borderColor: 'transparent',
```

Comment:
We can remove `border` and `borderColor` properties if we don't want the border. Right now it will still create a `1px` gap.